File: Beecrowd/1174/1174.py
# Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1174
arr = []
for i in range(0, 100):
    arr.append(float(input()))

for j in range(0, 100):
    if arr[j] <= 10.0:
        print(f'A[{j}] = {arr[j]}')

File: Beecrowd/1174/1174.c
#include <stdio.h>

int main(){
    float arr[100];
    for(int i = 0; i < 100; i++){
        scanf("%f", &arr[i]);
    }
    for(int j = 0; j < 100; j++){
        if(arr[j] <= 10.0){
            printf("A[%d] = %0.1f\n", j, arr[j]);
        }
    }

    return 0;
}

